### DISCUSSION TITLE
[Suggestion] Multiple toast types

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/flux#132] [Feature Requests] [answered]
Hello,

I'd like to suggest adding multiple toast types, like "success", "warning", "error" and "info". Each using different colors.

**Comment by jtesta66** [comment]
Agreed.  For now, I've just modified the published toast file.

**Comment by DonCamillo11** [comment]
It would be good to set easy color customizations for e.g. success or error toasts without having to publish the components.

**Comment by dicktornfeldt** [comment]
Also, would be nice to be able to add an icon. 

Somehting like:
`Flux::toast(message: 'Your changes have been saved.', variant: 'success', icon: 'check');`

**Comment by joshhanley** [comment]
Thanks for reporting! I have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

**Comment by dudelisius** [comment, marked as answer]
20 hours ago the latest version 1.0.9 was release and this supports variants; https://fluxui.dev/components/toast#variants

**Comment by mlewist** [comment]
Unfortunately the variant only seems to change 1 minor thing, adding an icon.

Ideally I want 'success', 'danger' etc to change the background colour of the toast as well (or even better, optionally allow me to).

I've published the toast view and copied over the variant classes used on the icons to work on background, but it seems a pretty clunky way to do it. 
Am I doing it wrong, is there a cleaner way?

`bg-white [[data-flux-toast-dialog][data-variant=danger]_&]:!bg-red-100 [[data-flux-toast-dialog][data-variant=info]_&]:!bg-blue-100 [[data-flux-toast-dialog][data-variant=success]_&]:!bg-lime-100  [[data-flux-toast-dialog][data-variant=warning]_&]:!bg-amber-100`

**Reply by timgavin** [reply]
See if this helps: https://github.com/livewire/flux/discussions/1211#discussioncomment-12344622
